src/component/common/ResponsiveButton/ResponsiveButton.tsx

```diff
@@ -2,6 +2,7 @@ import { useMediaQuery } from '@material-ui/core';
 import ConditionallyRender from '../ConditionallyRender';
 import PermissionButton from '../PermissionButton/PermissionButton';
 import PermissionIconButton from '../PermissionIconButton/PermissionIconButton';
+import React from 'react';
```

Comment:
Does this cause a tsx error? I thought this was implicit in react 17.

Reply:
Oh right!